collections/Ash_Rolls/MS_Ash_Rolls_38.xml

```diff
@@ -95,8 +95,8 @@
                   <adminInfo>
                      <recordHist>
                         <source>Description adapted (2018) from Watson and Pächt and Alexander (see bibliography), incorporating additional description of decoration by Elizabeth Solopova, c. 2000. Previously described in the Quarto Catalogue (W. H. Black, <title>A descriptive, analytical, and critical catalogue of the manuscripts bequeathed unto the University of Oxford by Elias Ashmole Esq....</title>, Quarto Catalogues X, 1845). <listBibl>
-                              <bibl facs="aeh0010.gif" type="QUARTO">Quarto Catalogue X, col. 10 (no. 23)</bibl>
-                              <bibl facs="aaq0486.gif" type="SC">Summary Catalogue, vol. 2, part 2, p. 1127</bibl>
+                              <bibl><ref target="https://hdl.handle.net/2027/uc1.31158010681335?urlappend=%3Bseq=15">Quarto Catalogues X, col. 10 (no. 23)</ref></bibl>
```

Comment:
Do you think colour or black-and-white scans are easier to read? See <https://hdl.handle.net/2027/mdp.39015041155527?urlappend=%3Bseq=13> for an alternative.

Reply:
probably black and white in this case?

Reply:
Yet another possibility would be to mix in Archive.org scans if available and of better quality: in this case <https://archive.org/details/descriptiveanaly00bodl/page/n10/>.